bot/exts/help_channels/_cog.py

```diff
@@ -114,6 +123,9 @@ async def claim_channel(self, message: discord.Message) -> None:
 
         await _caches.unanswered.set(message.channel.id, True)
 
+        # Removing the help channel from the dynamic message, and editing/sending that message.
+        self.available_help_channels.remove(message.channel.id)
```

Comment:
Why are we storing ids instead of the channels themselves?

Reply:
+1 to this. It will also make formatting the message more straightforward with channel.metion